packages/mobx-state-tree/src/core/node/node-utils.ts

```diff
@@ -10,8 +10,11 @@ import {
     AnyNode,
     IAnyType,
     IType,
-    assertArg
+    assertArg,
+    STNValue,
+    Instance
 } from "../../internal"
+import { IAnyComplexType } from "../type/type"
```

Comment:
seems like autoimport... we try to pass all imports throuth internal as far as I rememeber

Reply:
fixed